strr-examiner-web/app/layouts/examiner.vue

```diff
@@ -74,7 +73,7 @@ async function onChange (index: number) {
 
     <ConnectSystemBanner />
 
-    <main class="app-inner-container app-body bg-white">
+    <main class="app-inner-container app-body">
```

Comment:
@deetz99 no more bg-white? The header should have white bg, should I more the class there?